content/zh-cn/docs/quick-start.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "快速开始"
-description: "A quick start to know Clusternet"
-date: 2022-03-08
+标题: "快速开始"
```

Comment:
Please DO NOT translate the key here, such as `title`, `description`.

Reply:
fiexd